app/components/header/header.component.html

```diff
@@ -134,7 +134,8 @@
       </div>
 
       <ul class="nav navbar-nav visible-xs">
-        <li (click)="promptUserToAddRepository()"><a>&nbsp;&nbsp;add repository</a></li>
+        <!-- "add repository" is redundant -->
+        <!-- <li (click)="promptUserToAddRepository()"><a>&nbsp;&nbsp;add repository</a></li> -->
```

Comment:
If this is the case should this code just be removed?

Reply:
Sure if everyone agrees!